.eslintrc

```diff
@@ -9,7 +9,7 @@ env:
 
 rules:
   indent: [2, 2] # 2 spaces indentation
-  max-len: [2, 80, 4]
+  max-len: [2, 100, 4]
```

Comment:
That's why you need to confirm PR, not just merge. It's asking.
eslint doesn't like this in test/index.js. That string has 100 chars.
```
(t,
arg,
arg)
```